test/default_route_test.rb

```diff
@@ -101,9 +102,8 @@
     subject.Read
     subject.current = 0
 
-    builder = Y2Network::InterfaceConfigBuilder.new
+    builder = Y2Network::InterfaceConfigBuilder.for("eth")
```

Comment:
well I know that ```InterfaceConfigBuilder::for``` handles even strings, I'm just curious if using string here was an intention.

Reply:
well, I start on same places using `builder = Y2Network::InterfaceConfigBuilder.for(Y2Network::InterfaceType::ETHERNET)` but it is incredible long with all that scoping.